pattoo/api/portal/react/src/Agents.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import Navigation from './Navigation';
+import AgentsHead from './AgentsHead';
+import AgentsRow from './AgentsRow';
+
+class Agents extends React.Component {
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.